src/static/js/routes/offices/index.js

```diff
@@ -0,0 +1,24 @@
+/* ==========================================================================
+   Scripts for `/offices/`.
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var MobileOnlyExpandable = require( '../../modules/classes/MobileOnlyExpandable' );
+
+function init() {
+
+  // TODO: Remove this when per-page JS is introduced.
+  if ( document.querySelectorAll( '.office' ).length === 0 ) {
+    return;
+  }
+
+  var breakpointPx = 599;
+
+  $( '.expandable__mobile-only' ).each( function() {
+    new MobileOnlyExpandable( $( this ), breakpointPx ); // eslint-disable-line
```

Comment:
For some reason ESLint isn't allowing ignoring of individual rules, so a code comment above L:20 just explaining which rule this is intended to ignore would be great! Please add something like:
`// ESLint no-new rule ignored`

Eventually we'll want to re-write classes that require instantiation with new without assignment to a variable.